src/main/java/org/broadinstitute/hellbender/tools/walkers/RevertBaseQualityScores.java

```diff
@@ -23,7 +23,7 @@
         programGroup = ReadProgramGroup.class
 )
 
-public class RevertBaseQualityScores extends ReadWalker {
+public final class RevertBaseQualityScores extends ReadWalker {
```

Comment:
My 2 cents - I generally resist the temptation to do this on the assumption that we don't want to mix commits. Either way it would be good to have a policy one way or the other.